setup.py

```diff
@@ -0,0 +1,28 @@
+from setuptools import setup, find_packages
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(
+  name='yin-yang',
+  version='3.3.0',
+  url='https://github.com/oskarsh/Yin-Yang',
+  description='Auto Nightmode for KDE, Gnome, Budgie, VSCode, Atom and more',
+  long_description=long_description,
+  long_description_content_type="text/markdown",
+  packages=find_packages(),
+  classifiers=[
+      "Programming Language :: Python :: 3",
+      "License :: OSI Approved :: MIT License",
+      "Operating System :: OS Independent",
+  ],
+  python_requires='>=3.10',
+  install_requires=[
+    'psutil==5.9.5',
```

Comment:
Can you try to collect the dependencies in a single location? With these changes, we would have three different entries to change with an update: 
- requirements.txt
- setup.py and
- pyproject.toml. 

It would be great if we could just rely on one location.

Reply:
My understanding of these files is that requirements.txt is for development and setup.py/pyproject.toml is for packaging/distribution, so I think we need to have at least two files? It does appear that pyproject.toml can completely replace setup.py but it is still recommended that both pyproject.toml and setup.py stick around[^1], but I would appreciate feedback from someone who is more knowledgeable on python packaging then myself.

[^1]: https://packaging.python.org/en/latest/guides/modernize-setup-py-project/

Reply:
Actually, it appears that the modern recommendation is indeed to [forego setup.py in favor of pyproject.toml](https://setuptools.pypa.io/en/latest/userguide/quickstart.html#id2), if scripting is not needed. I'll work on that now.

Reply:
My latest commit sets up the project to use poetry for building, which means we only need to have pyproject.toml. If you are on board with switching to poetry, I will need to also update the build documentation since the commands will change.